_assets/js/netlify/topics/makeBodyContent.js

```diff
@@ -0,0 +1,14 @@
+const md = window.markdownit();
+md.options['html'] = true;
+md.options['linkify'] = true;
+const parser = new DOMParser();
+// This function takes our body content as input, turns the markdown into HTML,
+// then loops over the html elements
+const makeBodyContent = (bodyContent) => {
+  let html = parser.parseFromString(md.render(bodyContent), 'text/html');
+  let elements = html.children[0].children[1].childNodes;
```

Comment:
`html.children[0].children[1].childNodes` is a bit cryptic, and chains like this make me worried that elements may not exist (e.g., `html.children[0]` is often `head`)

can you add an implementation comment explaining what this should be/why it's guaranteed to be that?

Reply:
Does this make sense?

`The DOMParser returns a full HTML document, so grabbing the first child element actually grabs the <html> element because it is the first element under <!doctype> the second child element of the <html> is the <body> element. We aren't starting at the <html> element but rather the <!doctype> tag.`

Reply:
lgtm, thanks!